run.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 
 echo "npm run start$VERSION"
-npm run start$VERSION
+npm run buildFrontend
+npm run startBackend
```

Comment:
This will bypass the changes I'm working on to pass an environment variable. It will make it so the Template version will not work. Is it necessary to split these out instead of doing both from the package.json file?

Reply:
Really the issue is windows, but I don't forsee us using windows for deployment.  We can do the same thing in the package.json file with `&&` to run the commands in sequence.  Another option is to use `buildFrontend$Version` and create a template version as well.